dashboard/src/components/Config/AppRepoList/AppRepoList.v2.tsx

```diff
@@ -84,41 +130,53 @@ function AppRepoList({ cluster, namespace, kubeappsNamespace }: IAppRepoListProp
           {!errors.fetch && (
             <>
               <LoadingWrapper loaded={!isFetching}>
-                {/* TODO(andresmgot): Split between global and namespaced repositories */}
-                <Table
-                  valign="center"
-                  columns={[
-                    { accessor: "name", Header: "Name" },
-                    { accessor: "url", Header: "URL" },
-                    { accessor: "accessLevel", Header: "Access Level" },
-                    { accessor: "namespace", Header: "Namespace" },
-                    { accessor: "actions", Header: "Actions" },
-                  ]}
-                  data={repos.map(repo => {
-                    return {
-                      name: repo.metadata.name,
-                      url: repo.spec?.url,
-                      accessLevel: repo.spec?.auth?.header ? "Private" : "Public",
-                      namespace: repo.metadata.namespace,
-                      actions: (
-                        <AppRepoControl
-                          repo={repo}
-                          secret={repoSecrets.find(secret =>
-                            secret.metadata.ownerReferences?.some(
-                              ownerRef => ownerRef.name === repo.metadata.name,
-                            ),
-                          )}
-                          namespace={namespace}
-                          kubeappsNamespace={kubeappsNamespace}
-                        />
-                      ),
-                    };
-                  })}
-                />
+                <h3>Global Repositories</h3>
+                <p>
+                  Global repositories are available for all Kubeapps users.{" "}
+                  {namespace !== kubeappsNamespace && (
+                    <>
+                      Administrators can go to the{" "}
+                      <Link to={app.config.apprepositories("default", kubeappsNamespace)}>
+                        {kubeappsNamespace}
+                      </Link>{" "}
+                      namespace to manage them.
+                    </>
+                  )}
+                </p>
+                {globalRepos.length ? (
+                  <Table
+                    valign="center"
+                    columns={tableColumns}
+                    data={getTableData(globalRepos, namespace !== kubeappsNamespace)}
+                  />
+                ) : (
+                  <p>No global repositories found.</p>
+                )}
+                {namespace !== kubeappsNamespace && (
+                  <>
+                    <h3>Namespace Repositories</h3>
```

Comment:
I'm not sure about the terminology here. If you use the above, it should be "Namespaced Repositories", but I wonder if we can do better with `<h3>{namespace} Repositories</h3>`, which, next to "Global Repositories" would be clear, I think?

Reply:
The thing that bugs me is having a lowercase at the beginning: `<h3>default Repositories</h3>`, I'd rather do something like Namespace Repositories: default

Reply:
Yeah, it would look a bit odd, unless it had formatting for code or similar (ie. `<h3><code>{ namespace}</code> Repositories<h3>`). Your one is great too. Note that in this case it should be singular: `Namespace Repository: default` (unless the intention is that there's more than one there, as it was with the global repositories), but not a problem.

Reply:
mmh, in this case `default` is not the "namespace repository" what `Namespace Repositories: default` means is "namespaced repositories available in the default namespace" that's why I set it plural. Not sure if my English is right here?

Reply:
Oh right - yes, that's fine. I'd missed that we want to list multiple "namespaced" app repos for a single namespace, potentially. All good :)